recipes/mfast/all/patches/00005-add-conan-libs-to-targets.patch.txt

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/fast_type_gen/CMakeLists.txt b/src/fast_type_gen/CMakeLists.txt
+index 4e604cd..3762ea3 100644
+--- a/src/fast_type_gen/CMakeLists.txt
++++ b/src/fast_type_gen/CMakeLists.txt
+@@ -11,8 +11,7 @@ add_executable (fast_type_gen
+ target_link_libraries (fast_type_gen
+                        mfast_xml_parser_static
+                        mfast_static
+-                       ${Boost_SYSTEM_LIBRARY}
```

Comment:
I hope we can solve it soon with the components feature ... You needed to create a big patch only to solve a problem which should be solved by Conan generators ...

Reply:
You mean this feature:

https://docs.conan.io/en/latest/creating_packages/package_information.html#using-components

?

Reply:
https://github.com/conan-io/conan-center-index/pull/2097